v2/cf-nodejs-logging-support/src/lib/logger/root-logger.ts

```diff
@@ -1,15 +1,47 @@
 import Level from "./level"
 import Logger from "./logger"
+import Config from "../config/config"
+import { ConfigObject, customFieldsFormat } from "../config/interfaces"
+import EnvService from "../core/env-service";
 
 export default class RootLogger extends Logger {
+    private config = Config.getInstance();
 
     constructor() {
         super()
         this.loggingLevelThreshold = Level.INFO
     }
 
-    addConfig(_object: Object) {
-        // todo: add to config
+    getConfig() {
+        return this.config.getConfig();
+    }
+
+    getFields(...fieldNames: string[]) {
+        return this.config.getFields(fieldNames);
+    }
+
+    getMsgFields() {
```

Comment:
are these methods needed?
besides, i would vote for coming up with a different Name than msg fields, as we already call our customer message msg in our stacks, which would lead to confusions in the future.

Reply:
No, these methods (getDisabledFields, getMsgFields, getReqFields) should not be exposed. See also https://github.com/SAP/cf-nodejs-logging-support/pull/108#discussion_r836604071